README.md

```diff
@@ -119,47 +119,83 @@ print(output.shape)
 
 # Notes
 -> pairwise representation -> explicit atomic positions
+
 -> within the trunk, msa processing is de emphasized with a simpler MSA block, 4 blocks
+
 -> msa processing -> pair weighted averaging 
+
 -> pairformer: replaces evoformer, operates on pair representation and single representation
+
 -> pairformer 48 blocks
+
 -> pair and single representation together with the input representation are passed to the diffusion module
+
 -> diffusion takes in 3 tensors [pair, single representation, with new pairformer representation]
+
 -> diffusion module operates directory on raw atom coordinates
+
 -> standard diffusion approach, model is trained to receiev noised atomic coordinates then predict the true coordinates
+
 -> the network learns protein structure at a variety of length scales where the denoising task at small noise emphasizes large scale structure of the system.
+
 -> at inference time, random noise is sampled and then recurrently denoised to produce a final structure
+
 -> diffusion module produces a distribution of answers
+
 -> for each answer the local structure will be sharply defined
+
 -> diffusion models are prone to hallucination where the model may hallucinate plausible looking structures
+
 -> to counteract hallucination, they use a novel cross distillation method where they enrich the training data with alphafold multimer v2.3 predicted strutctures. 
+
 -> confidence measures predicts the atom level and pairwise errors in final structures, this is done by regressing the error in the outut of the structure mdule in training,
--> Utilizes diffusion rollout procedure for the full structure generation during training ( using a largeer step suze than normal)
+
+-> Utilizes diffusion rollout procedure for the full structure generation during training ( using a larger step suze than normal)
+
 -> diffused predicted structure is used to permute the ground truth and ligands to compute metrics to train the confidence head.
+
 -> confidence head uses the pairwise representation to predict the lddt (pddt) and a predicted aligned error matrix as used in alphafold 2 as well as distance error matrix which is the error in the distance matrix of the predicted structure as compared to the true structure
+
 -> confidence measures also preduct atom level and pairwise errors
+
 -> early stopping using a weighted average of all above metic
+
 -> af3 can predict srtructures from input polymer sequences, rediue modifications, ligand smiles
+
 -> uses structures below 1000 residues
```

Comment:
What is this comment referring to?
AFAIK, this only applies to their evaluation procedure against RF2NA.

Is this a limit decided for this implementation or does it refer to the above?